src/client/components/ProfileCareerResult/ProfileCareerResult.styles.css

```diff
@@ -32,12 +32,15 @@
 }
 
 .triangle {
+  margin-left: 0;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
   width: 0;
```

Comment:
I think, if you give the button position: relative and give it a little bottom space of i.e 5px, looks nicer. Now they are not inline :)